desc/coils.py

```diff
@@ -689,8 +704,6 @@ def from_values(cls, current, coords, N=10, NFP=1, basis="rpz", sym=False, name=
         sym : bool
             Whether to enforce stellarator symmetry.
         name : str
-            name for this coil
```

Comment:
Why delete this? I think it should still say ` Name for this coil.` like we have for all the similar methods of other coil classes.